_data/merchants.yml

```diff
@@ -118,24 +118,18 @@
       url: http://www.guitartheoryrevolution.info/blog/guitar-theory-revolution-store/
     - name: MyMonero Web-based Wallet
       url: https://mymonero.com
-    - name: Pradeep Atluri, Psychiatrist, New York
-      url: http://dr.mindsci.com/
     - name: Simple, no non-sense hosting
       url: https://rootbox.host/
     - name: Web Developer - Stefanos
       url: http://www.stefanosioannou.com/web-development-monero-accepted
     - name: XMR.to Monero to Bitcoin Payment Service
       url: https://xmr.to
-    - name: algoStrategic - Internet Marketing and Web Development
-      url: https://algostrategic.com
     - name: Emmanuel Galang, Canada Immigration and Refugee Lawyer
       url: https://galanglaw.com/
     - name: Web Developer - Python with Django web framework
       url: http://www.voteforrodneylewis.com
     - name: FlokiNET
       url: https://flokinet.is
-    - name: Nerdzy Lawn Care
-      url: http://www.nerdzy.net
```

Comment:
Looks like they're still up, still doing lawn care and still accepting Monero for payment.

http://www.nerdzy.net/index.php/lawn-care/

>Methods of payment: Our team believes in innovation and that is why we accept a variety of currencies at Nerdzy. As a whole, we accept: cash (USD), PayPal/Credit, Dogecoin (DOGE), Bitcoin (BTC), Monero/Bitmonero (XMR), and Ethereum (ETH)